Spectrum/src/com/unhurdle/spectrum/Banner.as

```diff
@@ -0,0 +1,75 @@
+package com.unhurdle.spectrum
+{
+    import org.apache.royale.core.UIBase;
+    COMPILE::JS{
+        import org.apache.royale.html.util.addElementToWrapper;
+        import org.apache.royale.core.WrappedHTMLElement;
+    }
+
+    public class Banner extends UIBase
+    {
+        public function Banner()
+        {
+            super();
+            typeNames = "spectum-Banner";
+            _color = "info";
+        }
+        COMPILE::JS
+        override protected function createElement():WrappedHTMLElement{
+            var elem:WrappedHTMLElement = addElementToWrapper(this,'div');
+            var header:WrappedHTMLElement = new WrappedHTMLElement();
```

Comment:
You can't use new WrappedHTMLElement. Use the newElement() function to create te element with the correct tag.